textbook/04/2/Dictionaries.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>Though in the previous  -&gt;  In the previous</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>